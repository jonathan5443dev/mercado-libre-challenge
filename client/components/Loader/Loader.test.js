import React from "react";
import Loader from "./Loader";

import renderer from "react-test-renderer";
import LightTheme from "@config/themes/light";
import { ThemeProvider } from "styled-components";

describe("Loader", () => {
  test("should render", () => {
    const component = renderer.create(
      <ThemeProvider theme={LightTheme}>
        <Loader color="secondary" />{" "}
      </ThemeProvider>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
