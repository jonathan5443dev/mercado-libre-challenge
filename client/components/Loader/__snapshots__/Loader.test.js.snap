// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should render 1`] = `
Array [
  <div
    className="sc-1g2dfmx-1 ixRDmH"
  >
    <div
      className="sc-1g2dfmx-0 ipxbwN"
      color="secondary"
    />
  </div>,
  " ",
]
`;
